store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReaderBuilder.java

```diff
@@ -83,6 +132,16 @@ public CarbonReaderBuilder projection(String[] projectionColumnNames) {
     return this;
   }
 
+  public CarbonReaderBuilder projection(List<String> projectionColumnNames) {
+    Objects.requireNonNull(projectionColumnNames);
```

Comment:
No need of this change ? It has already handled in #3197

Reply:
removed